backend/src/main/java/com/google/minicrm/data/Lead.java

```diff
@@ -25,8 +25,16 @@
 import java.util.Date;
 import java.util.HashMap;
 import java.util.List;
+import java.util.ArrayList;
+import java. util.*;
```

Comment:
You shouldn't import java.util.*; only import exactly what you need.

Reply:
Yea my bad